Add line in build-consumer to echo spin instance

elanalynn opened the pull request:
### Background

This is a tiny thing, but I feel more confident seeing where `build-consumer-spin` is copying the build.

### Solution

I added a line to echo the spin instance.

![echo spin](https://user-images.githubusercontent.com/7654369/145452012-5f2f690b-37bd-48e7-89aa-686b16f54f26.png)


### 🎩

Run `yarn build-consumer-spin web` and confirm that the output is your current spin instance.

### Checklist

- [x] I have :tophat:'d these changes
- [x] I have updated relevant documentation

vividviolet commented on scripts/consumer-helper.sh:
```diff
@@ -52,6 +52,7 @@ function copy_to_target {
     targetDir=$(resolve "$ROOT/../$projectDirectoryOrWorkspace/node_modules/@shopify/$package")
   else
     targetDir="$shopifyNodeModulesDir/$package"
+    echo "🌀 spin instance: `spin show | grep Shopify/$projectName | awk '{print $1}'` "
```
Instead of adding a new message here, can we add to line 27 instead? This is where it's saying whether we're running spin or not

vividviolet reviewed (commented):
Looks good - I have a minor suggestion of moving the message for simplicity

elanalynn commented:
@vividviolet I moved the spin echo up, and since we are using the url twice, I simplified and made it a variable. Also, since I'm now testing this on my M1, I had to update the node version.

vividviolet reviewed (approved):
Great job 👍
